CHANGELOG.md

```diff
@@ -1,5 +1,616 @@
 # Changelog
 
+## 12.0.0
+
+Teleport 12 brings the following marquee features and improvements:
+
+- Device Trust (Preview, Enterprise only)
+- Passwordless Windows access for local users (Preview, Enterprise only)
```

Comment:
Sorry to be a pedant, but we could access local users previously, what's changed is that we don't need integration with Active Directory to do it.

Reply:
- This is the approved language from marketing that we'll be using for the release. "Non AD" doesn't mean as much to new customers who don't know that we only supported AD to start with.
- No, we could never access local users previously, only AD enrolled users.